test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -1731,7 +1730,17 @@ protected Set<List<String>> indexRandomForMultipleSlices(String... indices) thro
                 refresh(index);
             }
         }
-        return bogusIds;
+        if (!bogusIds.isEmpty()) {
+            // delete the bogus types again - it might trigger merges or at least holes in the segments and enforces deleted docs!
```

Comment:
I know this is a copied code comment but I don't think `delete the bogus types again` makes sense here